### DISCUSSION TITLE
Filtering the file system

### DISCUSSION THREAD

**Discussion by LarryT8959** [nicolargo/glances#2569] [General]
I've installed glances on Ubuntu 22.04 and it starts automatically in server mode. It runs fine with no issues and I can view stats from any client on my network. I want to change the number of File Systems that are listed. Currently, there are about ten listed. All I want to see is the main drive, /dev/sda3. I've read through the manual an edited the fs section, saved the config file, but when I go to glances from the client again, I see the same info. It's like I didn't edit the file. I did go back to check to see if changes were saved, and they were. Here's a screen shot of what I'm getting currently.
<img width="502" alt="Screenshot 2023-09-27 at 7 12 56 PM" src="https://github.com/nicolargo/glances/assets/81517389/1f0e3aa0-eabf-4db5-bc2f-36a2abe970ef">

**Comment by nicolargo** [comment]
It is configurable from the [fs] section of the Glances configuration file.

More informatino here: https://glances.readthedocs.io/en/latest/aoa/fs.html
